guides/common/modules/proc_synchronizing-ansible-collectios.adoc

```diff
@@ -0,0 +1,39 @@
+:_module-type: PROCEDURE
+
+[id="proc_synchronizing-ansible-collectios_{context}"]
+= Synchronizing Ansible Collectios
+
+[role="_abstract"]
+On {Project}, you can synchronize your Ansible Collections from Automation Hub, cloud.redhat.com, and other {Project} instances.
+Ansible Collections will appear on {Project} as a new repository type in the {ProjectWebUI} menu under *Content*.
+
+NOTE: Ansible Collections live in the Library environment and the lifecyrcle management is not supported.
+Capsules with Pulp 3 have collections support and will sync collection repositories if tied to the Library environment.
+
+.Procedure
+
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Select the required product name.
+. In the *Products* window, select the name of a product that you want to create a repository for.
+. Click the *Repositories* tab, and then click *New Repository*.
+. In the *Name* field, enter a name for the repository.
++
+The *Label* field is populated automatically based on the name.
+. From the *Type* list, select *ansible collection*.
+. In the *Upstream URL* field, enter the URL for the upstream collections repository.
++
+The URL can be any Galaxy endpoint.
+. Optional: In the *Requirements.yml* field, you can specify the list of collections you want to sync from the endpoint, as well as their versions.
++
+For more information, see link:https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#install-multiple-collections-with-a-requirements-file[Install multiple collections with a requirements file] in the _Galaxy User Guide_.
+. Authenticate.
+.. To sync {Project} from *Automation Hub*, enter your token in the *Auth Token* field.
++
+For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/1.0/html-single/managing_red_hat_certified_and_ansible_galaxy_collections_in_automation_hub/index#proc-create-api-token[Retrieving your Red Hat Certified Collections Sync URL and API token] in the _Managing Red{nbsp}Hat Certified and Ansible Galaxy collections in Automation Hub_ guide.
+.. To sync {Project} from `cloud.redhat.com`, enter your token in the *Auth Token* field and enter your SSO URL in the the *Auth URL* field.
++
+For more information, see link:https://console.redhat.com/ansible/automation-hub/token[Connect Private Automation Hub] in the _Connect to Hub_ guide.
```

Comment:
Aren't the links reversed? I'd say this talks about the Private Automation Hub, and the first link is about the Hub on cloud.redhat.com